here/here_aggregation.md

```diff
@@ -46,7 +46,7 @@ Example of a table (from `activeto.analysis_periods`):
 
 | analysis_period | time_period | time_range | dow_range | day_type |
 | --- | --- | --- | --- | --- |
-| 1 | Weekday- Daily | [00:00:00,24:00:00) | [1,5] | Weekday |
+| 1 | Weekday- Daily | \[00:00:00,24:00:00) | [1,5] | Weekday |
```

Comment:
what duz this change do?

Reply:
It's a typo - I'll remove it. I didn't intend to change anything in the here_aggregation.md - I just opened it to see what was in it.